pages/projects.js

```diff
@@ -0,0 +1,76 @@
+import styles from '../components/landing_page.module.css'
+import s from '../components/project_list.module.css'
+
+function Projects() {
+  return (
+    <div className={s.body} id={s.bg}>
+      <div className={styles.body}>
+        <div className={styles.row}>
+          <div className={styles.column}>
+
+           <div className={styles.nav_bar}>
+              <a href ="/"><img src="/images/TempLogo.jpg" alt="Tufts Code For
+              Good Logo" width= "181px" height="61px" border="solid 1px #979797"/>
+              </a>
+              <a href="/about" className={styles.nav_bar}>ABOUT</a>
+              <a href="/projects" className={styles.nav_bar}>PROJECTS</a>
+              <a href="/funding" className={styles.nav_bar}>FUNDING</a>
+            </div>
+          </div>
+        </div>
+      </div>
+      
+        <div id={s.text_container}>
+          <h1 className={s.txt} id={s.title}>PROJECTS</h1>
+          <h3 className={s.txt} id={s.sub}>TUFTS CODE FOR GOOD</h3>
+        </div>
+        <div className={s.column}>
```

Comment:
For some reason the whole page doesn't scroll down for me. I'm not exactly sure what the issue is but I think it's a CSS problem with the main container. I'm looking into it.

Reply:
I changed the background's position value from 'relative' to 'absolute' and the scrolling is fixed now